packages/zcli-apps/tests/functional/mocks/single_product_ignore/.zcliignore

```diff
@@ -0,0 +1,2 @@
+assets/logo-small.png
+assets/iframe.html
```

Comment:
nit: needs to have new line at end of the file :)

Reply:
```
Patterns following the gitignore specification included in a .zcliignore file
```
I guess we should add more pattern from gitignore in order to test here :)

Reply:
Added more patterns : )